.github/workflows/master_cd.yml

```diff
@@ -6,21 +6,31 @@ on:
     branches:
       - master
 
+
+
 jobs:
   # Build DBC and Binaries
   build:
     name: Build Firmware Binaries
     runs-on: ubuntu-latest
+    env:
+      BUILD_DIR: build_fw_deploy
```

Comment:
I don't think you need to do this, because build_dir is default in the `build_binary` action. If you need to specify it, add it at L46.